src/view/components/typography/body.jsx

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React from "react";
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import getDimensionStyle from "../../utils/getDimensionStyle";
+
+/**
+ * Provides typography styling for a block of text (a paragraph).
+ * This is typically unnecessary, since the React-Spectrum Provider
+ * component provides basic default body styling for all content,
+ * but this can be useful in scenarios where a different font size
+ * is needed.
+ * See https://opensource.adobe.com/spectrum-css/typography-body.html
+ * for more info. Once React-Spectrum provides a Body component
+ * that handles typography, we can get rid of this.
+ */
+const Body = ({ children, size = "S", isSerif, marginTop, marginBottom }) => {
+  const style = {};
```

Comment:
Should we allow passing the whole style? too much?

Reply:
I think it would be best to wait until we need it. If we do end up needing it, I believe we should expose it as a prop named `UNSAFE_style`, because that's how React-Spectrum would do it, and I hope to replace this eventually with an actual React-Spectrum implementation. Typically, React-Spectrum provides enough props related to styling that you don't need to use the `UNSAFE_style` prop. You can see a list of all their typical styling props [here](https://react-spectrum.adobe.com/react-spectrum/View.html#props). To keep things simple in our implementation, I've only exposed `marginTop` and `marginBottom` from that list, because I think that's all we would typically need in this component.